src/redux/app/actions.ts

```diff
@@ -50,3 +51,10 @@ export const updateBlockchainOrder = (blockchains: IBlockchainsOptions[]) => {
         data: { blockchains }
     };
 };
+
+export const setBottomSheet = (type: BottomSheetType, blockchain?: Blockchain) => {
```

Comment:
and you can also create a `closeBottomSheet` function

Reply:
instead of using `blockchain?`, use a `props?` param of type `Object`. This way we can use this parameter for all bottom sheets that need parameters.

so your call will be like:
```
openBottomSheet(BottomSheetType.ACCOUNT_SELECTION, {blockchain: Blockchain.ETHEREUM});
```